config/configuration.json

```diff
@@ -3,8 +3,7 @@
     "scanUrls": [
       "org.openstreetmap.atlas.checks.validation"
     ],
-    "type": "org.openstreetmap.atlas.checks.base.BaseCheck",
-    "enabled.value.default": true
```

Comment:
This needs to be reverted.

Reply:
Can you check again for this? It looks like this was a part of one of my last commits

Reply:
It does look like line 7 was removed @reichg

Reply:
ah yes now I see, had to look at dev branch to see what it was supposed to look like haha

Reply:
So if you look under the "Files Changed" in the PR, it shows the comparison of your branch with the dev branch. So anything in red is deleted from dev and anything in green is newly added. :)